spring-cloud-stream/src/main/java/org/springframework/cloud/stream/function/StreamFunctionProperties.java

```diff
@@ -40,6 +40,8 @@
 
 	private String outputDestinationName = Processor.OUTPUT;
 
+	private boolean batchMode;
```

Comment:
I didn't notice that the `FunctionInvoker` has access to the `ConsumerProperties`. I think we should move this there; that way, the same flag will be used by both the invoker and the binder (if the binder supports batching).